### ISSUE TITLE
How to set output duration when using oneshot cross langual cloning?

### ISSUE THREAD

**Issue by lucasjinreal** [FunAudioLLM/CosyVoice#741]
How to set output duration when using oneshot cross langual cloning?

**Comment by MonolithFoundation** [comment]
I think it means the duration of output wav. It's a feature from many voice cloning system such as F5 or FishSpeech.

**Comment by gbtota** [comment]
Yes i wish this had duration parameter like F5 TTS does because the timing of the speech is very difficult thing to fix after generation

**Comment by LostnD** [comment]
say something? at least tell us how this works based on what It choses the duration output In voice cloning
